.travis.yml

```diff
@@ -6,6 +6,8 @@ node_js:
   - "6"
 before_install:
 - npm -g install npm@4
+- curl https://sh.rustup.rs -sSf | sh -s -- -y
+- source $HOME/.cargo/env
```

Comment:
This is going to be a problem. End users `npm install` the CLI on their own machines, often Windows machines, and they're not going to have Rust & Cargo installed. This (and the travis output) suggests that's not going to work, is that right?

I think we did discuss this briefly before, sorry I didn't call it out properly on the previous review. There's two options I think: hold this PR until we have CLI bundling completed (so users don't ever run `npm install` themselves), or set up node pre-gyp to precompile and publish reconfix as a binaries that users can use immediately.

The 2nd is probably more practical, if you've got time to set it up, and we'll want it if we want to use reconfix in any other node projects anyway, so we don't have to add a rust build env to every other project. We already do this for other native CLI packages like node-ext2fs. https://github.com/resin-io/node-ext2fs/pull/11 + https://github.com/resin-io/node-ext2fs/pull/13 are a full setup example I think, which should take you most of the way.

To get this working, you need:
* Versionbot set up (we should do this regardless)
* A CI build for each target platform/node version (see node-ext2fs)
* Enough Node configuration to build the project (we should have this already)
* A bit of npm boilerplate for node-pre-gyp & node-pre-gyp-release (see node-ext2fs)

The end goal is that on every tagged commit to master (from versionbot), a binary for every target platform & node version is built, and published to github releases. On every install of the package (including from CLI users), we look in github releases for a prebuilt binary, and use that immediately if there's a compatible one available (or try to build from scratch if not).

This is a non-trivial chunk of work, might be worth more discussion, feel free to ping me. @jviotti might have opinions on this too.

Reply:
Doing prebuilds is definitely an option (we usually favor `prebuild` rather than `node-pre-gyp` given the latter does some weird dynamic stuff that doesn't play nice with PKG) but proper PKG integration is something that would need to do anyways, right? How far are we with that? I remember seeing that you got this working at this point.

@abrodersen I can help you out setting this up for the Node.js bindings, but we should also do the PKG thing asap here.

Reply:
We have bundled packages published with all releases, but they don't work perfectly I think, and they're not well tested. It's in the readme, but marked as experimental. Needs finishing, but I've only really got time for quick & essential bug fixes on the CLI right now.